diff --git a/include/dmlc/base.h b/include/dmlc/base.h
index 085ca61..3d70e31 100644
--- a/include/dmlc/base.h
+++ b/include/dmlc/base.h
@@ -41,9 +41,9 @@
  * \brief Wheter to print stack trace for fatal error,
  * enabled on linux when using gcc.
  */
-#if (!defined(DMLC_LOG_STACK_TRACE) && defined(__GNUC__) && !defined(__MINGW32__))
-#define DMLC_LOG_STACK_TRACE 1
-#endif
+//#if (!defined(DMLC_LOG_STACK_TRACE) && defined(__GNUC__) && !defined(__MINGW32__))
+//#define DMLC_LOG_STACK_TRACE 1
+//#endif
 
 /*! \brief whether compile with hdfs support */
 #ifndef DMLC_USE_HDFS
